### PR TITLE
Add --nocolor and --forcecolor options

### LINKED ISSUE #78: Add --nocolor flag for plain text output
I mostly look at nox output in text files, in contexts that don't render ansi escape sequences.  So the output looks like this:

```
Step #7: ^[[36mnox > ^[[32mSession unit(py='3.6') was successful.^[[0m
Step #7: ^[[36mnox > ^[[33mRan multiple sessions:^[[0m
Step #7: ^[[36mnox > ^[[32m* unit(py='2.7'): success^[[0m
Step #7: ^[[36mnox > ^[[32m* unit(py='3.4'): success^[[0m
Step #7: ^[[36mnox > ^[[32m* unit(py='3.5'): success^[[0m
Step #7: ^[[36mnox > ^[[32m* unit(py='3.6'): success^[[0m
Step #7: ^[[36mnox > ^[[31m"Sessions not found: unit(py='2.7'), unit(py='3.4'), unit(py='3.5'), unit(py='3.6')"^[[0m
Step #7: ^[[36mnox > ^[[33mRunning session unit(py='2.7')^[[0m
Step #7: ^[[36mnox > ^[[34m/usr/bin/python -m virtualenv /home/vmagent/app/google-cloud-python/.nox/unit-2-7 -p python2.7^[[0m
Step #7: ^[[36mnox > ^[[36mchdir /home/vmagent/app/google-cloud-python/bigtable^[[0m
Step #7: ^[[36mnox > ^[[34mpip install --upgrade mock pytest pytest-cov ../api_core ../core^[[0m
Step #7: ^[[36mnox > ^[[34mpip install --upgrade -e .^[[0m
Step #7: ^[[36mnox > ^[[34mpy.test --quiet --cov=google.cloud.bigtable --cov=tests.unit --cov-append --cov-config=.coveragerc --cov-report= --cov-fail-under=97 tests/unit^[[0m
```

It would be nicer if there was a command line flag to inhibit escape sequences in the output.

### PR DESCRIPTION
**Pull request by jonparrott**
Default to nocolor when not running in a tty.

Resolves #78

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -104,6 +104,23 @@ By default nox will continue to run all sessions even if one fails. You can use
     nox --stop-on-first-error
 
 
+Controlling color output
+------------------------
+
+By default, Nox will output colorful logs if you're using in an interactive
+terminal. However, if you are redirecting ``stderr`` to a file or otherwise
+not using an interactive terminal, nox with output in plaintext.
```
Comment (dhermes) [inline docs/usage.rst]: s/with/will

File: nox/logger.py [hunk 2]
```diff
@@ -36,25 +36,33 @@ def success(self, msg, *args, **kwargs):
 logger.setLevel(logging.DEBUG)
 
 
-def setup_logging():  # pragma: no cover
+def setup_logging(color):  # pragma: no cover
+    """Setup logging.
+
+    Args:
+        color (bool): If true, the ouput will be colored using
```
Comment (dhermes) [inline nox/logger.py]: s/ouput/output

File: docs/usage.rst (hunk 1 above)
Comment (jonparrott) [reply docs/usage.rst]: Done

File: nox/logger.py (hunk 2 above)
Comment (jonparrott) [reply nox/logger.py]: Done

**Pull request merged**